rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -0,0 +1,589 @@
+      description: >
+        Applicable atomic energy information markings for a document or portion
+      type: string
+      example: RD
+    classification:
+      description: >
+        The highest level of classification applicable to the containing document or portion
+      type: string
+      example: U
+    classificationReason:
+      description: >
+        One or more reason indicators or explanatory text describing the basis for an original classification decision
+      type: string
+      example: Sensitive information crucial for victory
+    classifiedBy:
+      description: >
+        The identity, by name or personal identifier and position title, of the original classification authority for a document
+      type: string
+      example: Colonel Sanders
+    compilationReason:
+      description: >
+        The reason that the classification of the document is more restrictive than the simple roll-up of the marked portions of the document
+      type: string
+      example: TODO
+    compliesWith:
+      description: >
+        The ISM rule sets a document complies with
+      type: string
+      example: USGov
+    createDate:
+      description: >
+        The date when ISM metadata was added or updated
+      type: string
+      example: 20041117
+    declassDate:
+      description: >
+        The specific date when the resource is subject to automatic declassification procedures if not properly exempted from automatic declassification
+      type: string
+      example: 19960131
+    declassEvent:
+      description: >
+        A description of an event upon which the information shall be subject to automatic declassification procedures if not properly exempted from automatic declassification A description of an event upon which the information shall be automatically declassified if not properly exempted from automatic declassification.
+      type: string
+      example: End of conflict
+    declassException:
+      description: >
+        The exemption from automatic declassification that is claimed for a document
+      type: string
+      example: AEA
+    derivativelyClassifiedBy:
+      description: >
+        The identity, by name or personal identifier, of the derivative classification authority
+      type: string
+      example: The power rule derivatively classifies x^2 # TODO replace with a real example
+    derivedFrom:
+      description: >
+        A citation of the authoritative source or sources of the classification markings used in a derivative classification decision for a classified document
+      type: string
+      example: 2x is derived from x^2 # TODO replace with a real example
+    DESVersion:
+      description: >
+        The version number of the specification. This attribute is intended for processing systems to determine the appropriate versions of Schema, Schematron, and CVE values for validation and interpretation of an instance document.
+      type: string
+      example: TODO

```

Comment:
❓ Will you be able to get an example of these from someone. I just want to make sure they don't stay as TODOs. Same comments for these examples

Reply:
Ya, I need to fill these out eventually, and googling didn't prove to be very useful so they have to remain as TODOs for now 😞.